openslides/motions/static/js/motions/site.js

```diff
@@ -515,10 +528,35 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
                 });
         };
 
+        // Export the given motions as a csv file
+        $scope.csv_export = function (motions) {
```

Comment:
`motionsFiltered` is passed to the function. I originally thought of two export buttons: one for all motions and one for filtered motions. So different lists could be passed. If there isn't any need for it, i can change it, so two buttons ore one?
I would move it outside, if there is a need. Having the function in the controller makes it clear where it is used, if there is only one place.